db/postgres/export_from_mysql.py

```
@@ -0,0 +1,137 @@
+        "db": db,
+        "user": out["spring.datasource.username"],
+        "password": out["spring.datasource.password"],
+    }
+
+
+def query(cfg: dict, sql: str) -> list:
+    """탭 구분으로 받는다. NULL 은 \\N, 값 안의 탭·개행은 \\t \\n 으로 온다."""
+    result = subprocess.run(
+        ["mysql", "-N", "-B", "--default-character-set=utf8mb4",
+         "-h", cfg["host"], "-P", cfg["port"], "-u", cfg["user"],
+         f"-p{cfg['password']}", "--ssl-mode=REQUIRED", cfg["db"], "-e", sql],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)